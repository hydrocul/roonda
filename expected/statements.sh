perl $ROONDA_TMP_PATH/roonda_3c972d83a0d67a37fbe4537de63640edbf91039c.pl
ruby $ROONDA_TMP_PATH/roonda_76ced7bccb09d2de3e3de6cc0ad06af752f12041.rb
python2 $ROONDA_TMP_PATH/roonda_6ee585889d2fee0d0bbb96d3ab334b7ef4d222f6.py
python3 $ROONDA_TMP_PATH/roonda_6ee585889d2fee0d0bbb96d3ab334b7ef4d222f6.py
php $ROONDA_TMP_PATH/roonda_076630475537f0cd648552512af98b11614aa7f8.php

#################################################
# roonda_3c972d83a0d67a37fbe4537de63640edbf91039c.pl:
#################################################
# if (1) {
#     print "Hello ";
#     print "World!\n";
# } else {
#     print "ELSE\n";
# };

#################################################
# roonda_76ced7bccb09d2de3e3de6cc0ad06af752f12041.rb:
#################################################
# if 1
#   print "Hello "
#   print "World!\n"
# else
#   print "ELSE\n"
# end

#################################################
# roonda_6ee585889d2fee0d0bbb96d3ab334b7ef4d222f6.py:
#################################################
# import sys
# 
# if 1:
#     sys.stdout.write(str("Hello "))
#     sys.stdout.write(str("World!\n"))
# else:
#     sys.stdout.write(str("ELSE\n"))

#################################################
# roonda_076630475537f0cd648552512af98b11614aa7f8.php:
#################################################
# <?php
# 
# if (1) {
#     echo "Hello ";
#     echo "World!\n";
# } else {
#     echo "ELSE\n";
# };
