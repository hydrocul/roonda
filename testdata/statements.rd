sh v1

(sh
 (if (echo "a" (> "/dev/null"))
     ((print "Hello ")
      (print "World!\n"))
     ((print "ELSE\n")))
 (if (ls "/dummy" (> 2 "/dev/null"))
     ((print "TRUE\n"))
     ((print "OK\n")))
 (assign a 1))
(perl
 (if true
     ((print "Hello ")
      (print "World!\n"))
     ((print "ELSE\n")))
 (if false
     ((print "TRUE\n"))
     ((print "OK\n")))
 (assign a 1))
(ruby
 (if true
     ((print "Hello ")
      (print "World!\n"))
     ((print "ELSE\n")))
 (if false
     ((print "TRUE\n"))
     ((print "OK\n")))
 (assign a 1))
(python2
 (if true
     ((print "Hello ")
      (print "World!\n"))
     ((print "ELSE\n")))
 (if false
     ((print "TRUE\n"))
     ((print "OK\n")))
 (assign a 1))
(python3
 (if true
     ((print "Hello ")
      (print "World!\n"))
     ((print "ELSE\n")))
 (if false
     ((print "TRUE\n"))
     ((print "OK\n")))
 (assign a 1))
(php
 (if true
     ((print "Hello ")
      (print "World!\n"))
     ((print "ELSE\n")))
 (if false
     ((print "TRUE\n"))
     ((print "OK\n")))
 (assign a 1))
