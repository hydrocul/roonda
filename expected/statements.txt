Hello World!
OK
Hello World!
OK
Hello World!
OK
Hello World!
OK
Hello World!
OK
Hello World!
OK
